docs/streams/developer-guide/dsl-api.html

```diff
@@ -3153,11 +3159,182 @@ <h2><a class="toc-backref" href="#id7">Overview</a><a class="headerlink" href="#
+              <li>Better type safety during compile time.</li>
+            </ol>
+            <p>All functionality provided by Kafka Streams DSL for Scala are under the root package name of <code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala</span></code>.</p>
+            <p>Many of the public facing types from the Java API are wrapped. The following Scala abstractions are available to the user:</p>
+            <ul>
+              <li><code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala.StreamsBuilder</span></code></li>
+              <li><code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala.kstream.KStream</span></code></li>
+              <li><code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala.kstream.KTable</span></code></li>
+              <li><code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala.kstream.KGroupedStream</span></code></li>
+              <li><code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala.kstream.KGroupedTable</span></code></li>
+              <li><code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala.kstream.SessionWindowedKStream</span></code></li>
+              <li><code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala.kstream.TimeWindowedKStream</span></code></li>
+            </ul>
+            <p>The library also has several utility abstractions and modules that the user needs to use for proper semantics.</p>
+            <ul>
+              <li><code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala.ImplicitConversions</span></code>: Module that brings into scope the implicit conversions between the Scala and Java classes.</li>
+              <li><code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala.DefaultSerdes</span></code>: Module that brings into scope the implicit values of all primitive SerDes.</li>
+              <li><code class="docutils literal"><span class="pre">org.apache.kafka.streams.scala.ScalaSerde</span></code>: Base abstraction that can be used to implement custom SerDes in a type safe way.</li>
+            </ul>
+            <p>The library is cross-built with Scala 2.11 and 2.12.  To include it your maven <code>pom.xml</code> add the following:</p>
+            <pre class="brush: xml;">
+              &lt;dependency&gt;
+                &lt;groupId&gt;org.apache.kafka&lt;/groupId&gt;
+                &lt;artifactId&gt;kafka-streams&lt;/artifactId&gt;
+                &lt;version&gt;{{fullDotVersion}}&lt;/version&gt;
+              &lt;/dependency&gt;
+            </pre>
+            <p>If using SBT then you can add a reference to the library using the following:</p>
+            <pre class="brush: scala;">
+              libraryDependencies += "org.apache.kafka" %% "kafka-streams-scala" % "{{fullDotVersion}}"
```

Comment:
Don't we need the Scala version here?

Reply:
We don't need to specify the Scala version here.  The `%%` operator in SBT will automatically determine find the right artifact based on our running Scala version.